Update Chef Email & Password Under Account & Remove Refer A Chef Link

armandohimself opened the pull request:
# What Was Done

## Profile Page
1. Refactored profile page styling.
2. Refactored rendering of inputs.
3. Updating email updates in Authentication & Firestore.
4. Created modal to update password.
5. Error handling handles empty state and new passwords not matching.
6. Upon successful completion, user is taken back to profile page.
7. Updating password allows user to sign in with new password.

## Refer A Chef Link
1. Removed link in the account page.
2. Removed card on home page of chef dashboard that shows after account is completed.

armandohimself committed:
Updating Email in Auth & Firestore

Updating email happens in both places now.

armandohimself committed:
Added Change Password UI

armandohimself committed:
Finished Update Password Functionality

- Modal comes up and asks for new password with confirmed password. 
- Takes new password and updates user's password with new password
- After completion user is taken back to profile.

armandohimself committed:
Removed Refer A Chef

Removed in 2 places
1. In the chef account screen
2. Removed in the chef home screen when the account profile is finished

armandohimself committed:
Update TOS & PP 

- Added PP screen and added it to navigator
- TOS goes to EPC website
- Sign up now has both TOS and PP on page with updated component links.

JoshuaArcyLHarrison commented:
Can you go through all the files that you changes and format them. I believe prettier will use the `.editorconfig` to format them based on the project's settings.

armandohimself committed:
Update Format & Add Email GuestList Functionality

- Added button to email guest list on chef side
- Clicking button takes user to mailto client app and inserts event title as subject, their email as sender, and bcc's email guest list.

armandohimself committed:
Update Title

Updated event title and tested with more than 1 reserved guest.

armandohimself committed:
Fix Stash Mistake

JoshuaArcyLHarrison commented on versions/app/src/hooks/useSession.ts:
```diff
@@ -11,7 +11,7 @@ const useSession = () => {
 	const [loading, setLoading] = useState(true);
 	const [activeFlow, setActiveFlow] = useState(null); //chefs or guests
-	const [userID, setUserID] = useState("");
+	const [userID, setUserID] = useState(uid ? uid : "");
```
```suggestion
	const [userID, setUserID] = useState("");
```

JoshuaArcyLHarrison reviewed (changes requested):
Looks good. Just one change. You can click "Commit Suggestion -> Commit Changes"

armandohimself committed:
Update versions/app/src/hooks/useSession.ts

Co-authored-by: Joshua Arcy L. Harrison <89543882+Joshua-ArcyAdele@users.noreply.github.com>
